sig-docs/localization/ZH-CN Localization Task.md

```diff
@@ -0,0 +1,74 @@
+
+| Multi-cluster Management                                     | Translator | PR Link |
+| ------------------------------------------------------------ | ---------- | ------- |
+| [Index Page](https://kubesphere.io/docs/multicluster-management/) |            |         |
+| [Overview](https://kubesphere.io/docs/multicluster-management/introduction/overview/) |            |         |
+| [Kubernetes Federation in KubeSphere](https://kubesphere.io/docs/multicluster-management/introduction/kubefed-in-kubesphere/) |            |         |
+| [Direct Connection](https://kubesphere.io/docs/multicluster-management/enable-multicluster/direct-connection/) |            |         |
+| [Agent Connection](https://kubesphere.io/docs/multicluster-management/enable-multicluster/agent-connection/) |            |         |
+| [Retrieve KubeConfig](https://kubesphere.io/docs/multicluster-management/enable-multicluster/retrieve-kubeconfig/) |            |         |
+
+| Enable Pluggable Components                                  | Translator | PR Link |
+| ------------------------------------------------------------ | ---------- | ------- |
+| [Index Page](https://kubesphere.io/docs/pluggable-components/) |            |         |
+| [KubeSphere App Store](https://kubesphere.io/docs/pluggable-components/app-store/) |            |         |
+| [KubeSphere DevOps System](https://kubesphere.io/docs/pluggable-components/devops/) |            |         |
+| [KubeSphere Auditing Logs](https://kubesphere.io/docs/pluggable-components/auditing-logs/) |            |         |
+| [KubeSphere Events](https://kubesphere.io/docs/pluggable-components/events/) |            |         |
+| [KubeSphere Logging System](https://kubesphere.io/docs/pluggable-components/logging/) |            |         |
+| [KubeSphere Service Mesh](https://kubesphere.io/docs/pluggable-components/service-mesh/) |            |         |
+| [KubeSphere Alerting and Notification](https://kubesphere.io/docs/pluggable-components/alerting-notification/) |            |         |
+| [Network Policy](https://kubesphere.io/docs/pluggable-components/network-policy/) |            |         |
+
+| Upgrade                                                      | Translator | PR Link |
+| ------------------------------------------------------------ | ---------- | ------- |
+| [Index Page](https://kubesphere.io/docs/upgrade/)            |            |         |
+| [Overview](https://kubesphere.io/docs/upgrade/upgrade-overview/) |            |         |
+| [Upgrade with KubeKey](https://kubesphere.io/docs/upgrade/upgrade-with-kubekey/) |            |         |
+| [Upgrade with ks-installer](https://kubesphere.io/docs/upgrade/upgrade-with-ks-installer/) |            |         |
+| [Changes after Upgrade](https://kubesphere.io/docs/upgrade/what-changed/) |            |         |
+| [FAQ](https://kubesphere.io/docs/upgrade/upgrade-faq/)       |            |         |
```

Comment:
@Sherlock113 Are there any other chapters are ready to be translated?

Reply:
NO